velox/dwio/parquet/reader/PageReader.cpp

```diff
@@ -20,6 +20,7 @@
 #include "velox/dwio/common/BufferUtil.h"
 #include "velox/dwio/common/ColumnVisitors.h"
 #include "velox/dwio/parquet/thrift/ThriftTransport.h"
+#include "velox/dwio/parquet/writer/arrow/LevelConversion.h"
```

Comment:
If LevelConversion.h is used in both Reader and Writer, then maybe we should move it to common.

Reply:
Agree. Or `velox/dwio/parquet/arrow`.

Reply:
> Agree. Or `velox/dwio/parquet/arrow`.

Let's put it in common. We will gradually remove arrow code in the future, so don't want another arrow folder.

Reply:
I agree. Opened https://github.com/facebookincubator/velox/issues/11678. I started working on this.
I will open a PR today.